### ISSUE TITLE
Cannot recreate published results

### ISSUE THREAD

**Issue by vinitveersingh** [iMoonLab/MeshNet#14]
Hello @thss15fyt,
Thank you for your work and for making the code publicly available.
I trained your model and I'm not able to recreate the test accuracy of the pre-trained model. I'm getting a test accuracy of 0.8847 after 150 epochs and a training accuracy of 0.9054.

This issue has also been reported in #8, can you elaborate more on the following questions?
Can you tell me what was this complex training strategy or share its code?
Also, what problem in the data set are you referring to?

Thank you.

**Comment by thss15fyt** [comment]
Please refer to the new update and we provide a more stable training script to reproduce or even ourperform the results in paper.
